README.md

```diff
@@ -2,6 +2,19 @@
 
 a repo to practice clone, push, pull and handle merge conflicts.
 
+HEY ALL I NEED TO MAKE A PULL REQUEST TO DO SCREEN SHOTS FOR SOME WORK THING, ILL DELETE THIS WHEN IM DONE 
+
+BREAKING CHANGE
```

Comment:
Block: I think this line introduces a bug, if the variable comes in undefined it'll cause a red toast. Please check on this and fix